Dockerfile

```diff
@@ -825,30 +854,17 @@ LABEL org.opencontainers.image.source https://github.com/siderolabs/talos
 ENTRYPOINT ["/bin/installer"]
 
 FROM installer-image-squashed AS installer
-ARG TARGETARCH
-ENV TARGETARCH ${TARGETARCH}
-ONBUILD RUN apk add --no-cache --update \
-    cpio \
-    squashfs-tools \
-    xz
-ONBUILD WORKDIR /initramfs
-ONBUILD ARG RM
-ONBUILD RUN xz -d /usr/install/${TARGETARCH}/initramfs.xz \
-    && cpio -idvm < /usr/install/${TARGETARCH}/initramfs \
-    && unsquashfs -f -d /rootfs rootfs.sqsh \
-    && for f in ${RM}; do rm -rfv /rootfs$f; done \
-    && rm /usr/install/${TARGETARCH}/initramfs \
-    && rm rootfs.sqsh
-ONBUILD COPY --from=customization / /rootfs
-ONBUILD RUN xargs -a /usr/install/modules-${TARGETARCH}.txt -I {} install -D /rootfs/kernel/lib/modules/$(ls /rootfs/lib/modules)/{} /rootfs/lib/modules/$(ls /rootfs/lib/modules)/{} \
-    && depmod -b /rootfs $(ls /rootfs/lib/modules) \
-    && set -o pipefail && mksquashfs /rootfs rootfs.sqsh -all-root -noappend -comp xz -Xdict-size 100% -no-progress \
-    && find . 2>/dev/null | cpio -H newc -o | xz -v -C crc32 -0 -e -T 0 -z > /usr/install/${TARGETARCH}/initramfs.xz \
-    && rm -rf /rootfs /initramfs /rootfs/kernel
-ONBUILD WORKDIR /
 
 FROM installer-image-squashed AS imager
 
+# installer frontend
+FROM erichripko/simple-frontend AS installer-frontend
```

Comment:
maybe we add code from this to `hack`? It's not too crazy, and we can have full control over it (the upstream uses an old go toolchain)